src/vivarium_gates_nutrition_optimization_child/components/population.py

```diff
@@ -104,6 +109,22 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
     def _get_location(self, builder: Builder) -> str:
         return builder.data.load("population.location")
 
+    def modify_step(self, index: pd.Index) -> pd.Series:
+        """
+        Sets simlant step size to 0.5 for neonates and 4 for 1-5 months.
+        """
+        neonates = self.population_view.get(index, f"age < {NEONATAL_END_AGE}").index
+        one_to_five_mos = self.population_view.get(
+            index, f"age >= {NEONATAL_END_AGE} and age < 0.416667"
+        ).index
+        step_size = pd.concat(
+            [
+                pd.Series(pd.Timedelta(days=0.5), index=neonates),
```

Comment:
Shouldn't these Timedelta values be coming out of the model spec?

Reply:
If you mean that they are related to the min / standard step sizes somehow, I'd say not. These are hard-coded values specific to this "test run" implementation; but in principle you could make them configurable from the model spec ofc. In the long run, these values will be replaced with more general expressions / equations